doc/cephfs/mds-journaling.rst

```diff
@@ -55,3 +55,36 @@ line utility `cephfs-journal-tool` which can be used as follows:
 
 `cephfs-journal-tool` is also used to discover and repair a damaged Ceph File System.
 (See :doc:`/cephfs/cephfs-journal-tool` for more details)
+
+Journal Event Types
```

Comment:
What do you think about making a reference to the header file here?

Reply:
My concern with that is that it may lead to a developer inclined documentation rather than for the wider audience.